pkg/proc/proc.go

```diff
@@ -192,21 +173,11 @@ func (dbp *Process) Breakpoints() map[uint64]*Breakpoint {
 func (dbp *Process) LoadInformation(path string) error {
 	var wg sync.WaitGroup
 
-	exe, path, err := dbp.findExecutable(path)
-	if err != nil {
-		return err
-	}
-	fi, err := os.Stat(path)
-	if err == nil {
-		dbp.lastModified = fi.ModTime()
-	}
+	path = findExecutable(path, dbp.pid)
 
-	wg.Add(5)
+	wg.Add(1)
```

Comment:
`LoadBinaryInfo` actually spawns further goroutines and adds them to the WaitGroup.